docs/src/modules/components/AppLayoutDocs.js

```diff
@@ -92,7 +92,7 @@ function AppLayoutDocs(props) {
     productName = 'Material UI';
   }
   if (asPathWithoutLang.startsWith('/base')) {
-    productName = 'Base UI';
+    productName = 'MUI Base';
```

Comment:
I don't think that 1. is an option as it's not self-sustained: need to be able to search it on Google and find what you are looking for. For 2 vs. 3. No preferences, both could fly.

Reply:
With "MUI Core" and "MUI X" above and below, I think "MUI Base" would match better with consistency. 

p.s. I think renaming System to "MUI System" would make the names look more in order as well.

Reply:
Agree with @oliviertassinari - option 1 isn't the best. I don't have a strong preference. Also, since we decided not to follow the "MUI *" pattern for all the products, it opens up the possibility to rename Base to something more distinct.

Reply:
I'm with Michal, to not use too much time here to discuss the name and instead maybe focus on the installation walkthrough itself. We can better resolve naming by looking at everything at once holistically later.